parsers/CEB.py

```diff
@@ -40,14 +40,16 @@ def fetch_production(
     logger: Logger = getLogger(__name__),
 ):
     """Requests the previous day's production mix (in MW) for Sri Lanka, per quarter-hour"""
-    if target_datetime is not None:
-        raise NotImplementedError(
-            "It is not possible to fetch historical data for this datasource, it only has data for the previous day"
-        )
+    if target_datetime is None:
+        target_datetime = datetime.now(tz=timezone.utc)
```

Comment:
Did you take into account that the [data source](//cebcare.ceb.lk/gensum/details) only has data up to the previous day (inclusive)?

Reply:
Yep, when I tested it returns the most up to date data when it's the current date even if the current date doesn't exist yet.

We can do a time delta of - 1 day if you prefer that though.

Reply:
No, if it works it's fine.